pkg/operator/lockserver/params.go

```diff
@@ -33,15 +33,15 @@ const (
 
 // GlobalConnectionParams returns the Vitess connection parameters for a
 // VitessCluster's global lockserver.
-func GlobalConnectionParams(vt *planetscalev2.VitessCluster) *planetscalev2.VitessLockserverParams {
+func GlobalConnectionParams(lockSpec *planetscalev2.LockserverSpec, name string) *planetscalev2.VitessLockserverParams {
```

Comment:
Let's be more clear what name this refers to:

```suggestion
func GlobalConnectionParams(lockSpec *planetscalev2.LockserverSpec, clusterName string) *planetscalev2.VitessLockserverParams {
```

Reply:
Good idea, will update.

Reply:
Updated https://github.com/planetscale/vitess-operator/pull/33/commits/bc4749d003bca3d7843a076883ed0d9ce37495c4.